index.bs

```diff
@@ -1538,10 +1528,11 @@ partial interface mixin WindowOrWorkerGlobalScope {
 </xmp>
 
 Advisement: In Chrome this functionality is currently exposed as `FileSystemDirectoryHandle.getSystemDirectory({type: "sandbox"})`.
+This new method is available as of Chrome 85.
```

Comment:
I assume this means that the new name/semantics is in 85+ ? Maybe reword "is currently exposed" to "was exposed as .... in Chrome 84 and earlier." ?

Reply:
Changed it to "was previously exposed as" without explicitly saying when the old name went away, as it actually hasn't gone away yet, and will only be unavailable starting in M86/when we ship this outside of an origin trial. That hopefully is clear enough...